setup.py

```diff
@@ -53,6 +53,6 @@
         '': ['*.eml'],
         },
     test_suite='metaswitch.ellis.test',
-    install_requires=["py-bcrypt", "tornado==2.3", "msgpack-python", "phonenumbers", "SQLAlchemy", "MySQL-python"],
+    install_requires=["py-bcrypt", "tornado==2.3", "msgpack-python", "phonenumbers", "SQLAlchemy", "MySQL-python", "prctl"],
```

Comment:
Will this egg be automatically included in the Debian package, or might this break offline install?